com.richwhitehouse.BigPEmu.yaml

```diff
@@ -0,0 +1,41 @@
+id: com.richwhitehouse.BigPEmu
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: bigpemu
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.bigpemu_userdata
+  - --filesystem=host:ro
```

Comment:
this is a very broad filesystem permission for a *proprietary* emulator.

Reply:
- `device=all` to support controllers
- `persist=.bigpemu_userdata` built-in configuration directory
- `filesystem=home` needed for the application to create its built-in configuration directory
- `filesystem=host:ro` app opens roms/cd images that can be scattered in various filesystems
- `share=network` the app supports various networking modes
- `share=ipc` to ensure X11 performance
- `socket=fallback-x11` support X11
- `socket=pulseaudio` sound support
- `socket=wayland` video support
- `allow=bluetooth` for bluetooth controllers

Reply:
Yes, by default it creates its configuration directory in `~` home.  I've tried it both ways.  Without `filesystem=home` it won't create the directory and then won't be able to save settings within the application

Reply:
that's what `--persist` is for and you even use it.

Reply:
`--persist` alone doesn't seem to be enough to allow it to actually create the directory.  As mentioned above, I've tried it both ways, only with `--filesystem=home` does it create the directory